static/scripts/dataprivacy/registration-link-validation.js

```diff
@@ -5,7 +5,7 @@ window.addEventListener('DOMContentLoaded', () => {
 	const message = `
 	<div class="error-container">
 		<div class="error-header-wrapper">
-			<img class="error-header-image" src="/images/broken-pen.svg" alt="${$t('global.img_alt.brokenPen')}" />
```

Comment:
please also remove this i18n key (if not used any more) then we are good to go green :)